frontend/lib/identification/card_detail_view/extend_card_notification.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+
+import 'package:ehrenamtskarte/build_config/build_config.dart' show buildConfig;
+import 'package:ehrenamtskarte/l10n/translations.g.dart';
+import 'package:url_launcher/url_launcher_string.dart';
+
+class ExtendCardNotification extends StatefulWidget {
+  @override
+  State<ExtendCardNotification> createState() => _ExtendCardNotificationState();
+}
+
+class _ExtendCardNotificationState extends State<ExtendCardNotification> {
+  bool _isVisible = true;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context).colorScheme;
+    final t = context.t;
+
+    return _isVisible
+        ? Padding(
+            padding: const EdgeInsets.all(8.0),
+            child: Container(
+              padding: const EdgeInsets.all(16),
+              decoration: BoxDecoration(
+                color: theme.primary.withOpacity(0.1),
+                borderRadius: BorderRadius.circular(8),
+              ),
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Icon(Icons.info, color: theme.primary),
+                  SizedBox(width: 16),
+                  Expanded(
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Text(t.identification.extendCardNotificationTitle),
```

Comment:
applied, but I assume it depends on your PR https://github.com/digitalfabrik/entitlementcard/pull/1728
because currently the fonts don't match the design